tests/foreman/api/test_organization.py

```diff
@@ -349,9 +349,9 @@ def test_positive_add_smart_proxy(self):
         @CaseLevel: Integration
         """
         # Every Satellite has a built-in smart proxy, so let's find it
-        smart_proxies = entities.SmartProxy().search()
-        self.assertGreater(len(smart_proxies), 0)
```

Comment:
If default capsule doesn't exist, then search will return empty list and `.search()[0]` will lead to `'list index out of range` exception, so we won't miss the failure here. For me 'list index out of range` is pretty clear type of error, but it may be not that clear to everyone - asserting len of search results imo leaves no questions of why test failed, so maybe we should keep this assertion.